src/buffer.c

```diff
@@ -23,7 +23,7 @@ typedef struct BUFFER_TAG
 BUFFER_HANDLE BUFFER_new(void)
 {
     BUFFER* temp = malloc(sizeof(BUFFER));
-    /* Codes_SRS_BUFFER_07_002: [BUFFER_new shall return NULL on any error that occurs.] */
+    /* Codes_SRS_BUFFER_07_002: [If handle is NULL BUFFER_fill shall return a non-zero value.] */
```

Comment:
>/* Codes_SRS_BUFFER_07_002: [If handle is NULL BUFFER_fill shall return a non-zero value.] */ [](http://example.com/codeflow?start=4&length=93)

Looks like there is a missing spec for BUFFER_new which should read like
"If any error occurrs, BUFFER_new shall fail and return NULL."

And the previous author mistakenly tagged here a spec from BUFFER_fill.